models/camelot/camelot_pair_fee_rates.sql

```diff
@@ -0,0 +1,163 @@
+{{
+    config(
+        schema="camelot_arbitrum",
+        alias="pair_fee_rates",
+        partition_by=["blockchain"],
```

Comment:
this partition shouldn't be needed, as blockchain is always `arbitrum`

Reply:
should we change it to something like `pair`? or just remove it

Reply:
likely just remove it. the best option is typically a date column, such as `block_month` or something with relatively even distribution across partitions and enough data to impact performance. if you foresee querying this table and grouping by / filtering by month timeframes, that could help. otherwise, just remove